node.go

```diff
@@ -16,40 +17,51 @@ import (
 	"github.com/cosmos/iavl/internal/encoding"
 )
 
+// NodeKey represents a key of node in the DB.
+type NodeKey struct {
+	version int64
+	nonce   int32
+}
+
+func (nk *NodeKey) GetKey() []byte {
+	b := make([]byte, 12)
+	binary.BigEndian.PutUint64(b, uint64(nk.version))
+	binary.BigEndian.PutUint32(b[8:], uint32(nk.nonce))
+	return b
+}
+
 // Node represents a node in a Tree.
 type Node struct {
 	key           []byte
```

Comment:
If `nodeKey NodeKey` is now representing this node's key, what is the purpose of the `key` field?